### ISSUE TITLE
Trailing Newline Violation: Files should have a single trailing newline

### ISSUE THREAD

**Issue by ivanruizscm** [realm/SwiftLint#233]
<img width="913" alt="error" src="https://cloud.githubusercontent.com/assets/15628215/11361015/2cd94a88-928a-11e5-8422-94532b063f73.png">
I get this warning on every file at the end where i close the '}' from a class or struct.  
Whats its about? 

class Person {
    var name = ""
}  // Here i get the warning. Whats its suppoted to be?

**Comment by arsonik** [comment]
Just press enter at the end of your file ! (new line missing : \n)
or 
```
ed -s file.swift <<< w
```

**Comment by ivanruizscm** [comment]
And what about "Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace).
All functions are suppoted to be stick to each other?

<img width="985" alt="captura de pantalla 2015-11-24 a las 14 35 53" src="https://cloud.githubusercontent.com/assets/15628215/11368238/e39a6e7c-92b8-11e5-8550-b28b90359000.png">

**Comment by arsonik** [comment]
you are welcome !
------
you are not suppose (as its written in the yellow box) having whitespace on an empty line.
click on line 69 and you'll see that you can move the caret with left/right keypad
then delete the white space so the carte stick to the line number

**Comment by ivanruizscm** [comment]
Thx you! I know how to remove, but whats its the purpose of having functions without a blank line like so?
func test() {
}
func otherTest(){
}

This is fine?

**Comment by arsonik** [comment]
It's not about removing a line but spaces in line !!
"\n[space/tab]\n" must be "\n\n"
so you can still have 
```
func test() {
}

func otherTest(){
}
```
you should close this issue

**Comment by ivanruizscm** [comment]
Oh! i see now, thx
